### PR TITLE
Admin: Category - Project 2

### PR DESCRIPTION
**Pull request by phanlyhuynh**
Xem, thêm, sửa, xóa danh mục

### CODE REVIEW

**Comment by phanlyhuynh** [comment]
ready

File: resources/views/admin/category/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <div class="card-body">
+            <div class="row">
+                <div class="col-md-12">
+                    <div class="p-h-10">
+                        {!! Form::model($category, ['method'=>'PATCH', 'url' => 'admin/categories/' . $category->id, 'class' => 'm-t-15']) !!}
+                        <div class="form-group row">
+                            {!! Form::label(trans('home.category'), '', ['class' => 'col-sm-4 col-form-label control-label']) !!}
+                            <div class="col-sm-6">
+                                {!! Form::text('name', null, ['class' => 'form-control', 'placeholder' => 'Tên danh mục']) !!}
+                                {!! $errors->first('name','<p class="errors_validate">:message</p>') !!}
```
Comment (BunnyPi04) [inline resources/views/admin/category/edit.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_24-khoang-trang-whitespace-9
ý số 8

File: resources/views/admin/category/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+    <div class="card">
+        <div class="card-header border bottom">
+            <h4 class="card-title">{{ trans('home.edit_category') }}</h4>
+        </div>
+        <div class="card-body">
+            <div class="row">
+                <div class="col-md-12">
+                    <div class="p-h-10">
+                        {!! Form::model($category, ['method'=>'PATCH', 'url' => 'admin/categories/' . $category->id, 'class' => 'm-t-15']) !!}
```
Comment (BunnyPi04) [inline resources/views/admin/category/edit.blade.php]: convention
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_24-khoang-trang-whitespace-9
ý số 6

File: resources/views/admin/category/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                                </td>
+                            @endif
+                            <td class="text-center font-size-18">
+                                {!! Form::open(['method' => 'GET', 'url' => 'admin/categories/' . $category->id . '/edit']) !!}
+                                <button class="text-gray btn btn-float">
+                                    <i class="ti-pencil"></i>
+                                </button>
+                                {!! Form::close() !!}
+                                {!! Form::open(['method' => 'DELETE', 'url' => ['admin/categories', $category->id]]) !!}
+                                <button type="submit" class="text-gray btn btn-float" onclick="return confirm('{{ trans('home.confirm_delete') }}');">
```
Comment (BunnyPi04) [inline resources/views/admin/category/index.blade.php]: sử dụng laravel collective

**Comment by BunnyPi04** [comment]
commented

**Comment by phanlyhuynh** [comment]
ready

**Pull request merged**